### PR TITLE
Security info (tokens, artifacts, logs)

### PR DESCRIPTION
**Pull request by karensawrey**
Adding:

- API Access Token lifecycle and security
- Encryption and security in Pipelines/Managing Log Output and /Artifacts

### CODE REVIEW

**Comment by karensawrey** [comment]
The "update emojis" commit is the automatic aftermath of fixing a merge conflict. Please look into it and just ignore/reject it as necessary,

File: pages/pipelines/artifacts.md.erb [hunk 1]
```diff
@@ -64,6 +64,20 @@ The `buildkite-agent artifact download` command only works within the context of
...
 
+## Encryption and security  
+
+Buildkite has zero access to your source code in the pipelines and only receives and stores the log output of the builds and build artifacts in encrypted form. 
+
+!!!
+
+Logs are AES-encrypted, and the build artifacts are encrypted in transit and at rest using AWS encryption (KMS or S3 SSE). As a result, the keys cannot be extracted by an insider, and the AWS solutions provide mitigations against zero-day attacks and other security issues. Beyond this, the control over security measures within your infrastructure is up to you.
+
+If you choose to [host your own build artifacts](/docs/agent/v3/cli-artifact#using-your-private-aws-s3-bucket), they never leave your infrastructure and end up in your private AWS bucket.
```
Comment (plaindocs) [inline pages/pipelines/artifacts.md.erb:75]: Is it normal to consider your AWS as "your infrastructure"?

**Review by plaindocs — COMMENTED** [review]
This great to me, I've got a few minor queries and suggestions inline, but I'm excited to get this out.

File: pages/apis/managing_api_tokens.md.erb [hunk 2]
```diff
@@ -80,4 +80,12 @@ If you own the token, you can revoke your token from the [API Access Token page]
...
 ### What happens if I remove the access for a token that's currently in use?
 
-The token will lose access to the organization data. Any future API requests will no longer successfully authorize.
+The token will lose access to the organization data. Any future API requests will no longer successfully authorize.  
+
+## API Access Token lifecycle and security  
+
+API Access Tokens are issued for users within a Buildkite organization. They are stored both in the Buildkite database (linked to the user ID) and with the user for which they're issued.
+
+API Access Tokens are associated with a specific user and can only be revoked by that user. However, admins of a Buildkite organization can remove a user from an organization, and the removed user won't be able to access any organization's resources (like pipelines) using the existing API Access Token.
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply pages/apis/managing_api_tokens.md.erb:89]: Absolutely, thanks!

File: pages/pipelines/artifacts.md.erb (hunk 1 above)
Comment (karensawrey) [reply pages/pipelines/artifacts.md.erb:75]: Addressed this in the update.

File: pages/pipelines/artifacts.md.erb [hunk 3]
```diff
@@ -64,6 +64,20 @@ The `buildkite-agent artifact download` command only works within the context of
 
 If you want to download an artifact from outside a build use our [Artifact Download API](/docs/agent/v3/cli-artifact#downloading-artifacts-outside-a-running-build).
 
+## Encryption and security  
+
+Buildkite has zero access to your source code in the pipelines and only receives and stores the log output of the builds and build artifacts in encrypted form. 
+
+!!!
```
[the remark this answers is not in the document]
Comment (karensawrey) [reply pages/pipelines/artifacts.md.erb:71]: Fixed.

**Pull request merged**